build-ps/debian/percona-server-common-5.7.postinst

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+set -e
+
+case "$1" in
+  configure)
+    # New packaging paradigm for my.cnf as of Dec-2014 for sharing mysql
+    # variants in Ubuntu. If the new mysql-common package does not provide
+    # the update-alternatives facility, fall back to creating a symlink
+#    if [ -f /usr/share/mysql-common/configure-symlinks ]
+#    then
+#      /usr/share/mysql-common/configure-symlinks install percona-server "/etc/mysql/percona-server.cnf"
+#    else
+#      echo "Notice: configure-symlinks trigger could not be called."
+#      echo "Please manually make /etc/mysql/my.cnf a symlink that points to percona-server.cnf."
+#    fi
+    if [ ! -f /etc/mysql/my.cnf ]
+    then
+      update-alternatives --force --install /etc/mysql/my.cnf my.cnf "/etc/mysql/percona-server.cnf" 200
+    else
+      echo " -------------"
+      echo "   *  The suggested my.cnf options and settings are in percona-server.conf.d/mysqld.cnf"
```

Comment:
should this be: "* The suggested my.cnf options and settings are in /etc/mysql/percona-server.conf.d/mysqld.cnf"?
I would suggest to change the start to something like "* The suggested mysql server options and ..." because it lowers the confusion.